src/Home.tsx

```diff
@@ -613,66 +613,6 @@ const Home = (props: HomeProps) => {
                     }
                     clusterUrl={rpcUrl}
                     cluster={cluster}
-                    handleTransaction={async (transaction: Transaction) => {
-                      setIsUserMinting(true);
-                      const userMustSign = transaction.signatures.find((sig) =>
-                        sig.publicKey.equals(wallet.publicKey!)
-                      );
-                      if (userMustSign) {
-                        setAlertState({
-                          open: true,
-                          message: "Please sign one-time Civic Pass issuance",
-                          severity: "info",
-                        });
-                        try {
-                          transaction = await wallet.signTransaction!(
-                            transaction
-                          );
-                        } catch (e) {
-                          setAlertState({
-                            open: true,
-                            message: "User cancelled signing",
-                            severity: "error",
-                          });
-                          // setTimeout(() => window.location.reload(), 2000);
-                          setIsUserMinting(false);
-                          throw e;
-                        }
-                      } else {
-                        setAlertState({
-                          open: true,
-                          message: "Refreshing Civic Pass",
-                          severity: "info",
-                        });
-                      }
-                      try {
-                        await sendTransaction(
-                          props.connection,
-                          wallet,
-                          transaction,
-                          [],
-                          true,
-                          "confirmed"
-                        );
-                        setAlertState({
-                          open: true,
-                          message: "Please sign minting",
-                          severity: "info",
-                        });
-                      } catch (e) {
-                        setAlertState({
-                          open: true,
-                          message:
-                            "Solana dropped the transaction, please try again",
-                          severity: "warning",
-                        });
-                        console.error(e);
-                        // setTimeout(() => window.location.reload(), 2000);
-                        setIsUserMinting(false);
-                        throw e;
-                      }
-                      await onMint();
-                    }}
                     broadcastTransaction={false}
```

Comment:
we can also remove the broadcastTransaction flag as it's now the default

Reply:
sure, will remove